File: ts-sdk/src/sdk/models/operations/deleteadatabase.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class DeleteADatabasePathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=name" })
  name: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class DeleteADatabaseRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteADatabasePathParams;
}

export class DeleteADatabaseResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-sdk/pkg/models/operations/deleteadatabase.go
package operations

type DeleteADatabasePathParams struct {
	Name         string `pathParam:"style=simple,explode=false,name=name"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type DeleteADatabaseRequest struct {
	PathParams DeleteADatabasePathParams
}

type DeleteADatabaseResponse struct {
	ContentType string
	StatusCode  int
}
